[BUG]: Project debug build gives an crash error on each build

### Checked for duplicates?

- [X] This issue is not a duplicate

### Does it also happen in the desktop version?

- [ ] This bug does not occur in the latest version of Anki

### What are the steps to reproduce this bug?

Build the project -> Launch the app

### Expected behaviour

No error

### Actual behaviour

Log: 
```
Caused by: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
                                                                                                        at androidx.appcompat.app.AppCompatDelegateImpl.createSubDecor(AppCompatDelegateImpl.java:902)
                                                                                                        at androidx.appcompat.app.AppCompatDelegateImpl.ensureSubDecor(AppCompatDelegateImpl.java:865)
                                                                                                        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:748)
                                                                                                        at androidx.appcompat.app.AppCompatDialog.setContentView(AppCompatDialog.java:99)
                                                                                                        at androidx.appcompat.app.AlertController.installContent(AlertController.java:232)
                                                                                                        at androidx.appcompat.app.AlertDialog.onCreate(AlertDialog.java:278)
                                                                                                        at android.app.Dialog.dispatchOnCreate(Dialog.java:461)
                                                                                                        at android.app.Dialog.show(Dialog.java:339)
                                                                                                        at com.ichi2.anki.analytics.AnkiDroidCrashReportDialog.onCreate(AnkiDroidCrashReportDialog.kt:57)
```

### Debug info

```text
On main branch
```


### (Optional) Anything else you want to share?

_No response_

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

--- a/AnkiDroid/src/main/java/com/ichi2/anki/analytics/AnkiDroidCrashReportDialog.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/analytics/AnkiDroidCrashReportDialog.kt
@@ -17,12 +17,12 @@
 package com.ichi2.anki.analytics
 
 import android.annotation.SuppressLint
+import android.app.AlertDialog
 import android.content.DialogInterface
 import android.os.Bundle
 import android.view.View
 import android.widget.CheckBox
 import android.widget.EditText
-import androidx.appcompat.app.AlertDialog
 import androidx.core.content.edit
 import com.ichi2.anki.CrashReportService
 import com.ichi2.anki.R

--- a/lint-rules/src/main/java/com/ichi2/anki/lint/rules/AvoidAlertDialogUsage.kt
+++ b/lint-rules/src/main/java/com/ichi2/anki/lint/rules/AvoidAlertDialogUsage.kt
@@ -21,6 +21,7 @@ import com.android.tools.lint.client.api.UElementHandler
 import com.android.tools.lint.detector.api.*
 import com.google.common.annotations.VisibleForTesting
 import com.ichi2.anki.lint.utils.Constants
+import com.ichi2.anki.lint.utils.LintUtils
 import org.jetbrains.uast.UElement
 import org.jetbrains.uast.UImportStatement
 
@@ -57,7 +58,9 @@ class AvoidAlertDialogUsage : Detector(), SourceCodeScanner {
         return object : UElementHandler() {
             override fun visitImportStatement(node: UImportStatement) {
                 val importReference = node.asSourceString()
-                if (importReference.contains("android.app.AlertDialog")) {
+                if (importReference.contains("android.app.AlertDialog") &&
+                    !LintUtils.isAnAllowedClass(context.uastFile!!.classes, "AnkiDroidCrashReportDialog")
+                ) {
                     context.report(
                         ISSUE,
                         node,

